public/apps/configuration/panels/user-list.tsx

```diff
@@ -224,7 +224,7 @@ export function UserList(props: AppDependencies) {
               <EuiFlexItem>{actionsMenu}</EuiFlexItem>
               <EuiFlexItem>
                 <EuiButton fill href={buildHashUrl(ResourceType.users, Action.create)}>
-                  Create internal user
```

Comment:
The more I look at this change, the more I feel that we should leave this as `user` in place of `account`. But that could just be me, thoughts @samuelcostae @cwperks @scrawfor99

Reply:
I dont have a strong opinion, as I think either leaving/changing would be fine.  I think it is ok with the commited change as it is in sync with "Service Accounts", but again, not a strong opinion.

Reply:
Either is fine.